nemo_curator/core/client.py

```diff
@@ -127,6 +127,8 @@ def stop(self) -> None:
         if self.ray_process:
             self.ray_process.kill()
             self.ray_process.wait()
+            # reset the ray address
+            os.environ.pop("RAY_ADDRESS", None)
```

Comment:
if we always pop, then would this scenario fail

```python
ray start --port 1234
export RAY_ADDRESS=1234

python run.py # this relies on RAY_ADDRESS therefore doesn't start a new ray start because of line 96/97

python run.py # since RAY_ADDRESS is popped it'll start a new cluster
```

Reply:
Wait, I think if the ray_process is not set, why will this env var get popped ? For your case prateek, the ray_process is always none right?